test/e2e/reloadinterval_test.go

```diff
@@ -58,12 +58,11 @@ func TestReloadInterval(t *testing.T) {
 	}{
 		// set ReloadInterval to a value that does not pass the regex validation
 		// expect validation error
-		{"set reloadInterval 0.5s", metav1.Duration{Duration: 500 * time.Millisecond}, "5s", false},
```

Comment:
this was moved below and renamed `set reloadInterval 500ms` because it's actually `500ms`.